src/App.jsx

```diff
@@ -1,13 +1,156 @@
-
 import './App.css'
+import Button from './components/ButtonComponent.jsx'
+import {useState} from 'react'
+import Counter from "./components/CounterComponent.jsx";
+import TextInput from "./components/TextInputComponent.jsx";
+import RadioGroup from "./components/RadioGroupComponent.jsx";
+import Checkbox from "./components/CheckboxComponent.jsx";
 
 function App() {
 
-  return (
-    <>
-        <h1>Fruitmand bezorgservice</h1>
-    </>
-  )
+    //useState hooks voor elk type fruit
+    const [bananas, setBananas] = useState(0);
+    const [strawberries, setStrawberries] = useState(0);
+    const [apples, setApples] = useState(0);
+    const [kiwis, setKiwis] = useState(0);
+
+    //Functies om de hoeveelheid aan te passen
+    const increase = (setter, current) => setter(current + 1);
+    const decrease = (setter, current) => setter(Math.max(0, current - 1));
+
+    //Functie om alles te resetten
+    const reset = () => {
+        setBananas(0);
+        setStrawberries(0);
+        setApples(0);
+        setKiwis(0);
+    };
+
+    //Opdracht 2: Formulier bouwen
+    const [firstName, setFirstName] = useState('');
+    const [lastName, setLastName] = useState('');
+    const [age, setAge] = useState('');
+    const [postcode, setPostcode] = useState('');
+    const [deliveryFrequency, setDeliveryFrequency] = useState('iedere week');
+    const [timeSlot, setTimeSlot] = useState('overdag');
+    const [comment, setComment] = useState('');
+    const [termsAccepted, setTermsAccepted] = useState(false);
+
+    //Functie voor het versturen van het formulier
+    const handleSubmit = (e) => {
+        e.preventDefault(); //Voorkomt het herladen van de pagina
+        console.log({
+            firstName,
+            lastName,
+            age,
+            postcode,
+            deliveryFrequency,
+            timeSlot,
+            comment,
+            termsAccepted,
+            bananas,
+            strawberries,
+            apples,
+            kiwis
+        });
+    };
+    return (
+        <div className="fruit-container">
```

Comment:
Blijf letten op het gebruik van semantische elementen! Structuur in de HTML is nu afwezig.